x-pack/plugins/security_solution/public/detections/components/callouts/missing_privileges_callout/translations.tsx

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiCode } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React from 'react';
+import {
+  DetectionsRequirementsLink,
+  SecuritySolutionRequirementsLink,
+} from '../../../../common/components/links_to_docs';
+import {
+  DEFAULT_ITEMS_INDEX,
+  DEFAULT_LISTS_INDEX,
+  DEFAULT_SIGNALS_INDEX,
+  SAVED_OBJECTS_MANAGEMENT_FEATURE_ID,
+} from '../../../../../common/constants';
+import { CommaSeparatedValues } from './comma_separated_values';
+import { MissingPrivileges } from './use_missing_privileges';
+
+export const MISSING_PRIVILEGES_CALLOUT_TITLE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.missingPrivilegesCallOut.messageTitle',
+  {
+    defaultMessage: 'Insufficient privileges',
+  }
+);
+
+const CANNOT_EDIT_RULES = i18n.translate(
+  'xpack.securitySolution.detectionEngine.missingPrivilegesCallOut.cannotEditRules',
+  {
+    defaultMessage: 'Without that privilege you cannot create/edit detection engine rules.',
+  }
+);
+
+const CANNOT_EDIT_LISTS = i18n.translate(
+  'xpack.securitySolution.detectionEngine.missingPrivilegesCallOut.cannotEditLists',
+  {
+    defaultMessage: 'Without that privilege you cannot create/edit value lists.',
+  }
+);
+
+const CANNOT_EDIT_ALERTS = i18n.translate(
+  'xpack.securitySolution.detectionEngine.missingPrivilegesCallOut.cannotEditAlerts',
+  {
+    defaultMessage: 'Without that privilege you cannot change alert states (open or close alerts).',
+  }
+);
+
+export const missingPrivilegesCallOutBody = ({
```

Comment:
I could be wrong here, but I think in general the pattern in Kibana has been to keep the `.translation.ts` files to be just the translations and stuck `<FormattedMessage></>` within the components or in utils or something if it needs to be reusable.

Reply:
Hm 🤔 I've looked at the other callouts; it looks like they all use `<FormattedMessage />` inside `translations.tsx`. Like here: https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/public/detections/components/callouts/need_admin_for_update_callout/translations.tsx
So I also decided to place them here for consistency.

Reply:
Ah, nice, hadn't seen that one. 👍🏽